artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTSubscriptionManager.java

```diff
@@ -369,7 +373,7 @@ Map<Long, Integer> getConsumerQoSLevels() {
 
    void clean() {
       for (MqttTopicSubscription mqttTopicSubscription : session.getState().getSubscriptions()) {
-         removeSubscription(mqttTopicSubscription.topicName());
+         removeSubscription(mqttTopicSubscription.topicName(), true);
```

Comment:
The `testSubscriptionQueueRemoved` test is failing, should enforceSecurity=false in this case?
```suggestion
         removeSubscription(mqttTopicSubscription.topicName(), false);
```

Reply:
Yes! I changed that for a quick test and forgot to change it back.